bin/cli.js

```diff
@@ -94,7 +94,11 @@ function invoke(env) {
       'environment, default: process.env.NODE_ENV || development'
     )
     .option('--esm', 'Enable ESM interop.')
-    .option('--specific [path]', 'Specify one seed file to execute.');
+    .option('--specific [path]', 'Specify one seed file to execute.')
+    .option(
```

Comment:
Should this be a root level parameter?

Reply:
I think not. But I put the new parameter where the "--specific" was, and this one only affect seeds too.

Reply:
I tested it now, if I remove the parameter from root level, it will not be shown on the help command.